### ISSUE TITLE
3.2 Validation of EO Data - charts colour scale (enhancement)

### ISSUE THREAD

**Issue by lehotskak** [EO-College/cubes-and-clouds#70]
In the exercise 3.2, there are four error matrices. All error matrices use the same colour scale, but the values on the scales are different.
While this thing isn't crucial, it is against best practices in proper chart design and makes the charts more difficult to interpret.

It's better to use different colour scales for different value ranges.

The issue was found here - https://hub.eox.at/user/5a5bae1c-f1ef-4bbf-992b-cc58d2260501/lab/tree/32_validation.ipynb
![32_validation ipynb_error_matrices](https://github.com/user-attachments/assets/3af772ad-7170-478c-b731-ff22a77f3a07)

**Comment by clausmichele** [comment]
@lehotskak what do you think of this color scheme proposed by @jzvolensky? You can check the updated notebook here: https://github.com/EO-College/cubes-and-clouds/blob/070c52bf3c1b59ae894f0554439f78bc5d9c2d16/lectures/3.2_validation/3.2_exercises/32_validation.ipynb
![Untitled](https://github.com/user-attachments/assets/be6190da-03a6-49ba-9a75-64b8aa4873ab)

**Comment by lehotskak** [comment]
Hi Michele,

These look much better now. The two matrices on the bottom still have a bit
similar colour schemes (I checked it on a mobile phone only), but it's a
major improvement compared to the original version.

I love using this tool to find a colour scheme for a chart:
https://python-graph-gallery.com/color-palette-finder/

Kind regards
Karolína

**Comment by jzvolensky** [comment]
Hi @lehotskak 

Thank you for the feedback!

Very cool tool. I was actually struggling to find colors that match nicely while also being different enough. 
I will try to use the palette finder and put together a nicer combination 😄 .

**Comment by lehotskak** [comment]
Yes, finding the right colours is often the most difficult part. I am happy
you like the tool and I'll be looking forward to seeing the updates
matrices 😊
